pkg/gofr/datasource/file/fs.go

```diff
@@ -61,3 +61,26 @@ func (fileSystem) RemoveAll(path string) error {
 func (fileSystem) Rename(oldname, newname string) error {
 	return os.Rename(oldname, newname)
 }
+
+func (fileSystem) Stat(name string) (FileInfo, error) {
+	return os.Stat(name)
+}
+
+func (fileSystem) CurrentDir() (string, error) {
+	return os.Getwd()
+}
+
+func (fileSystem) ChangeDir(dir string) error {
+	return os.Chdir(dir)
+}
```

Comment:
Overloading the methods name seems strange. I assume you are reusing the one defined in FTP code

I know it's because of an interface, but I would have added a comment about it